packages/terra-toggle/src/terra-dev-site/doc/example/AnimatedToggle.jsx

```diff
@@ -19,7 +19,10 @@ class ToggleDefault extends React.Component {
     const { isOpen } = this.state;
     return (
       <div>
-        <IconInformation onClick={this.handleOnClick} />
+        <IconInformation onClick={this.handleOnClick} aria-expanded={isOpen} />
+        {/**
+        * The aria-expanded state is used on the triggering component to indicate the contents are collapsible, and whether a region is currently expanded or collapsed
+        */}
         <Toggle isOpen={isOpen} isAnimated>
```

Comment:
Let's add the id for the aria-controls mapping:
```suggestion
        <Toggle id="toggleContainer" isOpen={isOpen} isAnimated>
```

Reply:
Resolved in 3478f79